### PR TITLE
bug 1550028: Switch to crash-stats.mozilla.org

### PR DESCRIPTION
**Pull request by jwhitlock**
Switch the preferred production domain from crash-stats.mozilla.com to .org in code and docs.

### CODE REVIEW

File: docs/crashstorage/elasticsearch.rst [hunk 1]
```diff
@@ -18,7 +18,7 @@ web app. Here is a list of those features:
   queries. It is also accessible via the public API, allowing users to build
   their own tools.
 
-  Example: https://crash-stats.mozilla.com/search/
```
Comment (willkg) [inline docs/crashstorage/elasticsearch.rst]: I don't understand what this example is. We could remove this line.

File: docs/crashstorage/elasticsearch.rst [hunk 2]
```diff
@@ -36,14 +36,14 @@ web app. Here is a list of those features:
   share a signature. Features include listing crash reports, listing user
   comments and showing aggregation on any field of the database.
 
-  Example: https://crash-stats.mozilla.com/signature/?signature=nsTimeout::~nsTimeout%28%29
```
Comment (willkg) [inline docs/crashstorage/elasticsearch.rst]: We can remove this line. Seems like unhelpful maintenance burden to have examples of urls in the app.

File: docs/crashstorage/elasticsearch.rst [hunk 3]
```diff
@@ -36,14 +36,14 @@ web app. Here is a list of those features:
...
+  Example: https://crash-stats.mozilla.org/signature/?signature=nsTimeout::~nsTimeout%28%29
 
 * **Profile page**
 
   On the profile page, Super Search is used to show the recent crash
   reports that contain the user's email address.
 
-  Example: https://crash-stats.mozilla.com/profile/
+  Example: https://crash-stats.mozilla.org/profile/
```
Comment (willkg) [inline docs/crashstorage/elasticsearch.rst]: We can remove this, too.

**Comment by jwhitlock** [comment]
After removing the ``Example:`` lines, it was clear that the ElasticSearch Features section could be further simplified. It had the feel of a product announcement rather than a long term document. I've re-written the Features section, so it is less complex and less self-promoting.

**Comment by willkg** [comment]
Yeah, the Elasticsearch docs aren't helpful. I was thinking of doing a pass on Elasticsearch docs when I get back to working on Elasticsearch upgrade, but feel free to do a pass if you like.

**Pull request merged**